public/docs/ganache/using.md

```diff
@@ -0,0 +1,64 @@
+# Working with Ganache
+
+[Ganache](/ganache) is an application that generates a personal blockchain you can use to run tests, execute commands, deploy contracts, and inspect state.
```

Comment:
"an application that generates" - stronger if just "is". 

i.e., 

> [Ganache](/ganache) is a personal blockchain for Ethereum development you can use to deploy contracts, develop your applications, and run tests. 

I put "run tests" last as that's less of the focus of Ganache, although possible.